examples/integrations/gcp/src/jobs/etl/images/bounding_box/bounding_box_etl.py

```diff
@@ -61,7 +69,12 @@ def upload_to_gcs(image_bytes: BytesIO, data_row_uid: str, h: int, w: int,
     return f"gs://{bucket.name}/{blob.name}"
 
 
-def process_label(label: Label, bucket: Bucket) -> str:
+def process_label(label: Label, partition: Optional[Partition],
+                  bucket: Bucket) -> str:
```

Comment:
nit: I'd prefer to set the bucket type as "storage.Bucket" instead of importing Bucket and using it directly, because it is more descriptive. Also it's the way the text etl does it...

Reply:
yeah definitely